db/dataset_eval.py

```diff
@@ -273,7 +283,9 @@ def add_dataset_eval_set(connection, data):
     return snapshot_id
 
 
-def _create_job(connection, dataset_id, normalize, eval_location, filter_type=None):
+def _create_job(connection, dataset_id, normalize, eval_location, c_value, gamma_value,
```

Comment:
maybe we should consider using default values for the arguments `c_value`, `gamma_value` and `preprocessing_values`, that would mean that we don't have to change all the tests.